app/grandchallenge/components/models.py

```diff
@@ -2099,6 +2107,8 @@ def create_civ_for_image(self, *, ci, current_civ, new_value, user):
                     immutable=True,
                 )
             )
+        else:
+            raise RuntimeError("Unexpected value presented")
```

Comment:
If new_value is an image and it it the same as the current value, we do nothing. That needs to be accounted for here, or else we'd get errors in those cases.